### DISCUSSION TITLE
How does the A+A option look on an actual 1440 or 4k display?

### DISCUSSION THREAD

**Discussion by MattGamingV1** [bloc97/Anime4K#235] [Q&A]
At present I have a 1080P VA panel display but im hoping to upgrade to a 1440p IPS soon and I was wondering if A+A still has that over aliasing problem at 1440p. I use my pc mostly for gaming but got a 6700xt a few months ago and it cant really stretch its legs at 1080p ultra. Im just curious if A+A will be the better option at 1440p (or 4k depending on if I get a secondary non gaming display later as well)

**Comment by yooooooLeon** [comment]
您的邮件已收到！谢谢！

**Comment by Tama47** [comment]
I'm using Mode A (HQ) on a 65" 4K TV. I feel like it is the most perfect as A+A is just too sharp on a 4K screen.
